### PR TITLE
fix: multi fixes

### LINKED ISSUE #132: bug: incorrect icon naming
**Describe the bug**

We saw that the icon name between the github and the figma is different, especially for “chevron” icon. The name on github is “ic_vtmn_arrow_drop_right_fill.xml” and the same one on figma is “chevron” with “right” in direction properties. Here the two links to compare:

- [Github](https://github.com/Decathlon/vitamin-android/blob/main/foundation/foundation-icons/src/main/res/drawable/ic_vtmn_arrow_drop_right_fill.xml)  

- [Figma](https://www.figma.com/file/ujn50nNMfGKzZJccATv7Vk/Vitamin---Vitamix-icons-(v0)?node-id=18%3A15)

### PR DESCRIPTION
**Pull request by charlottemougenot**
## Changes description

- Change values of semantics border colors
- Change icon names for chevron icons 
- Update color for dividers
- Update color for secondary button border 

## Context

Closes #132 and #130 

## Checklist

- [X] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request directly from your main!
- [X] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
- [X] Check your code additions will fail neither code linting checks.
- [X] I have reviewed the submitted code.
- [X] I have tested on a phone device/emulator.
- [X] I have tested on a tablet device/emulator.
- [X] I have tested on a large screen device/emulator.
- [X] If it includes design changes, please ask for a review with a core team designer.

## Does this introduce a breaking change?

- Yes
Change the name of chevron icons from : 
### Change values of semantics border colors : 
**Light :** 
vtmnBorderColorPrimary : vtmn_white to vtmn_grey200
vtmnBorderColorSecondary : vtmn_grey200 to vtmn_grey100
vtmnBorderColorTertiary : vtmn_grey100 to vtmn_white

**Dark :** 
vtmnBorderColorPrimary : vtmn_grey800 to vtmn_grey700
vtmnBorderColorSecondary : vtmn_grey700 to vtmn_black
vtmnBorderColorTertiary : vtmn_black to vtmn_grey800
vtmnBorderColorPositive : vtmn_green400 to vtmn_conifer400

### Change icon names for following icons
| Old icon name                   | New icon name                        |
|---------------------------------|--------------------------------------|
| `ic_vtmn_arrow_down_s_line.xml` | `ic_vtmn_chevron_down_line.xml`      |
| `ic_vtmn_arrow_left_s_line.xml` | `ic_vtmn_arrow_chevron_left_line.xml`|
| `ic_vtmn_arrow_right_s_line.xml`|`ic_vtmn_arrow_chevron_right_line.xml`|
| `ic_vtmn_arrow_up_s_line.xml`   | `ic_vtmn_arrow_chevron_up_line.xml`  |
| `ic_vtmn_bar_chart_1_line.xml`  | `ic_vtmn_bar_chart_line.xml`         |
| `ic_vtmn_camera_3_line.xml`     | `ic_vtmn_camera_line.xml`            |
| `ic_vtmn_database_2_line.xml`   | `ic_vtmn_database_line.xml`          |
| `ic_vtmn_delete_bin_7_line.xml` | `ic_vtmn_delete_bin_line.xml`        |
| `ic_vtmn_file_3_line.xml`       | `ic_vtmn_file_line.xml`              |
| `ic_vtmn_focus_3_line.xml`      | `ic_vtmn_focus_line.xml`             |
| `ic_vtmn_gift_2_line.xml`       | `ic_vtmn_gift_line.xml`              |
| `ic_vtmn_heart_3_line.xml`      | `ic_vtmn_heart_line.xml`             |
| `ic_vtmn_message_2_line.xml`    | `ic_vtmn_message_line.xml`           |
| `ic_vtmn_mic_vtmn_line.xml`     | `ic_vtmn_mic_line.xml`               |
| `ic_vtmn_scales_3_line.xml`     | `ic_vtmn_scales_line.xml`            |
| `ic_vtmn_settings_3_line.xml`   | `ic_vtmn_settings_line.xml`          |
| `ic_vtmn_store_3_line.xml`      | `ic_vtmn_store_line.xml`             |
| `ic_vtmn_t_shirt_2_line.xml`    | `ic_vtmn_t_shirt_line.xml`           |

### Remove following icons
- `ic_vtmn_arrow_drop_down_line.xml`
- `ic_vtmn_arrow_drop_left_line.xml`
- `ic_vtmn_arrow_drop_right_line.xml`
- `ic_vtmn_arrow_drop_up_line.xml`

## Screenshots
No design changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ com.decathlon.vitamin.chips | Build UIs with ready to use Chip components.
 com.decathlon.vitamin.prices | Build UIs with ready to use Price component.
 com.decathlon.vitamin.dividers | Build UIs with ready to use Divider component.
 com.decathlon.vitamin.tags | Build UIs with ready to use Tag components.
+com.decathlon.vitamin.tabs | Build UIs with ready to use Tabs components.
```
Comment (lauthieb) [inline README.md:79]: Could be nice to have alphabetical order.

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/62ac2962b2dd627a/build/fc6bc62c-c4fa-410d-98b2-cae5f2b11eb5/artifact/f438a7fe69aeb739/p/ad5f3835b3de1a4fb835faf0dc26b9bb

**Review by GerardPaligot — APPROVED** [review]

**Comment by ravenfeld** [comment]
Hello, I see a name change for the chevron icon but the icon according to the figma is not correct, there is too much border and is therefore resized smaller.
Is it possible to update them at the same time?

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/62ac2962b2dd627a/build/dc775658-fa44-4dae-a1f4-efde49ec3791/artifact/ba7335d446d1546a/p/f2d81dc174fa0f27886aae0b195a5e12

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/62ac2962b2dd627a/build/ec8ef062-a84e-4433-baa3-39e3769a3472/artifact/93e54c7ea01aa7e1/p/e6a9312f5f640933284a72847da57cf9

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/62ac2962b2dd627a/build/aba582bd-6b07-4f51-a04c-4ed876252aa0/artifact/8ee4c5a631ce464d/p/09b24593ef103da3e2017b8f9305bd79

**Review by GerardPaligot — APPROVED** [review]

**Pull request merged**